graphs/eacs/src/test/kotlin/com/anaplan/engineering/azuki/graphs/eacs/BEH8.kt

```diff
@@ -0,0 +1,67 @@
+package com.anaplan.engineering.azuki.graphs.eacs
+
+import com.anaplan.engineering.azuki.core.runner.Eac
+import com.anaplan.engineering.azuki.core.system.BEH
+import com.anaplan.engineering.azuki.graphs.adapter.api.GraphBehaviours
+import com.anaplan.engineering.azuki.graphs.adapter.api.GraphFunctionalElements
+import com.anaplan.engineering.azuki.graphs.dsl.GraphScenario
+import com.anaplan.engineering.azuki.graphs.graphA
+
+
+@BEH(GraphBehaviours.CreateDirectedGraph, GraphFunctionalElements.DirectedGraph, """
+    Create a directed graph
+""")
+class BEH8 : GraphScenario() {
+
+    @Eac("When a graph is first initialised it contains no edges or vertices")
+    fun newGraph() {
+        given {
+            thereIsADirectedGraph(graphA)
+        }
+        then {
+            hasVertexCount(graphA, 0)
+        }
+    }
+
+    @Eac("A graph may contain any number of vertices")
+    fun withVertices() {
+        given {
+            thereIsADirectedGraph(graphA) {
+                vertex("a")
+                vertex("b")
+                vertex("c")
+            }
+        }
+        then {
+            hasVertexCount(graphA, 3)
+        }
+    }
+
+    @Eac("A graph may contain any number of edges")
+    fun withEdges() {
+        given {
+            thereIsADirectedGraph(graphA) {
+                edge("a", "b")
+                edge("b", "c")
+                edge("d", "e")
+            }
+        }
+        then {
+            hasVertexCount(graphA, 5)
```

Comment:
⛏️ For a more meaningful scenario, I would have liked to see multiple edges between the same set of vertices, and a check along the lines of `hasEdgeCount`. I realise however that we don't yet have that check, so you can leave it like this for now.